.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
+    - cd daemon-scheduler; make; make release; cd ../
```

Comment:
I do not. If each line in that section gets it's own shell, we wouldn't need `cd ../`. Also, the [build matrix](https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix) is something that could be of help here. Here's a [blog](https://lord.io/blog/2014/travis-multiple-subdirs/) that illustrates the same.